PusherServer/PusherOptions.cs

```diff
@@ -39,6 +42,25 @@ public bool Encrypted
         }
 
         /// <summary>
+        /// The host of the HTTP API endpoint excluding the scheme e.g. api.pusherapp.com
+        /// </summary>
+        /// <remarks>
+        /// If the scheme is included when setting the value it will be stripped from the set value.
+        /// e.g. setting with a value of "https://api.pusherapp.com" will result in a value of "api.pusherapp.com".
+        /// </remarks>
+        public string Host
+        {
+            get
+            {
+                return _host;
+            }
+            set
+            {
+                _host = Regex.Replace(value, "^https?://", string.Empty);
```

Comment:
There's already a `Port` option. A cleaner option may be throwing an exception if the supplied string isn't just a host name?

I'll maybe have a look at the `PusherOptions` object and try and clean this up more generally. Maybe a `Uri` property and an `Encrypted` bool...

I need to fix the backwards compatibility problem with `event_ids` being expected so that'll be an opportunity to look at this too.